server/config/toml.go

```diff
@@ -147,6 +147,18 @@ enable = {{ .GRPC.Enable }}
 # Address defines the gRPC server address to bind to.
 address = "{{ .GRPC.Address }}"
 
+###############################################################################
+###                        gRPC Web Configuration                           ###
+###############################################################################
+
+[grpc-web]
+
+# GRPCWebEnable defines if the gRPC-web should be enabled.
```

Comment:
This needs to make note of the fact that gRPC must also be enabled, otherwise, this configuration is a no-op.